### ISSUE TITLE
Does Not Send Mail Anymore

### ISSUE THREAD

**Issue by ajtouchstone** [pupi1985/IPMonitor#17]
Somewhere the mailer stopped working.  On the latest version of Java (8 U321 32/64).  When I run Test mail it says:

DEBUG: setDebug: JavaMail version 1.5.5

And nothing else...  Help?  Thanks!

**Comment by pupi1985** [comment]
I've just tested it and it's working fine. Double check your email settings.

**Comment by ajtouchstone** [comment]
Before I created this post, I completely removed the entire IP Monitor program, deleted all files and the service, then extracted a fresh copy of the program and configured the options correctly for my SMTP server.

**Comment by pupi1985** [comment]
99% of the email issues are related to misconfigurations on the client-side or changes on the server-side. I don't want to spend time asking questions about firewalls or allowing less secure apps in GMail. I'm sure you'll eventually figure out where the issue is. Feel free to update the post afterwards.

**Comment by ajtouchstone** [comment]
No, I've made no changes to the firewall and I am not going to use GMail.  I have a paid SMTP server service that is working flawlessly for dozens of other computers on the same network.  I have other services using Java on this server and suspected that was the problem.  Thanks for your info.

**Comment by mikecwwells** [comment]
Hi! I think OP may be on to something, as I encountered this same issue on Linux using Java 17 that was resolved by switching versions to Java 8. When testing the mail feature under Java 17, the console spit out a loooong error, ending with something like "classnotfoundexception javax.activation.datahandler". Manually starting the app using Java 8 sends the emails just fine. I haven't tried any other versions.
